app/services/tock_importer.rb

```diff
@@ -6,6 +6,7 @@ def perform
       tock_project = ClientAccount.find_or_initialize_by(tock_id: project["id"])
       tock_project.name = project["name"]
       tock_project.billable = project["billable"]
+      tock_project.live = project["live"]
```

Comment:
eg: look for a specific client account that should be active according to fixtures and make sure that on our side it is `active` too